handbook.md

```diff
@@ -752,12 +753,13 @@ The goal of Atlantian Youth Rapier is to recreate pre-17th century fencing.  All
             * Any approach specifically approved by the Deputy Earl Marshal for Rapier Combat.
 #. Conventions of Combat for Rapier Combat
     #. Valid Blow Definitions
-        #. In Rapier and Light Rapier, valid blows are thrusts, draw cuts, and push cuts.
-        #. In Cut & Thrust, valid blows are thrusts, draw cuts, push cuts, and percusive cuts.
+        #. In Rapier and Light Rapier, valid blows are thrusts and draw cuts.
+        #. In Cut & Thrust, valid blows are thrusts, draw cuts, and percusive cuts.
+        #. The only valid blow with a Rapier Spear is a thrust.
         #. Thrusts must be delivered using the tip, with positive force towards the opponent in line with the blade.
         #. Cuts must be delivered with positive force with the edge to be valid.
```

Comment:
With a sword that displays an edge and is not rounded.  Swords that have no edge should not be allowed to deliver cuts.  I.E. Italianated Foils, darkwood armories.

Reply:
I believe the intent of your proposal is to focus C&T exclusively towards weapons where cuts where a primary use case in period.  I have concerns about the wording.  As worded, this would prevent the use of multiple swords that are specifically designed for C&T style combat such as [Darkwood's Destreza Training Sword](http://www.darkwoodarmory.com/?main_page=product_info&products_id=366) as it has a square edge and [Castille Armory's Feder](https://castillearmory.com/product-category/mark-v-federschwert/) as it has a rounded edge.  Additionally, this prevents the use of reduced cost equipment [Darkwood's Rapier Foil](http://www.darkwoodarmory.com/index.php?main_page=product_info&cPath=11_59_12&products_id=375) while allowing an expensive alternative that performs similarly in practice [Castille Armory's Elite Rapier Blade](https://castillearmory.com/product/elite-rapier-blade/).

This differentiation could be discussed as Cut with Thrusts rather than Thrusts with Cuts.

While I can see value in enabling the highlight of cut-focused combat, I am hesitant to place additional restrictions in policy preventing forms of combat that do not measurably increase risk.  It is my feeling this is something that can be best done by participants and marshals.

Reply:
Do we see enough of those DW in C&T to care? I don't think I have experienced it, but I also think the marshals running stuff can limit their tournaments to edged simulators only or whatever. Excluding is OK.